src/qibocal/auto/operation.py

```diff
@@ -45,6 +52,16 @@ def load(cls, parameters):
 class Data:
     """Data resulting from acquisition routine."""
 
+    @staticmethod
+    def to_npz(path, data_dict: dict):
+        """Helper function to use np.savez while converting keys into strings."""
+        np.savez(path / DATAFILE, **{str(i): data_dict[i] for i in data_dict})
+
+    @staticmethod
+    def to_json(path, data_dict: dict):
```

Comment:
```suggestion
    def to_npz(path, data: dict):
        """Helper function to use np.savez while converting keys into strings."""
        np.savez(path / DATAFILE, **{str(i): data_dict[i] for i in data_dict})

    @staticmethod
    def to_json(path, data: dict):
```

`_dict` is redundant, it's already in the type.

Why do you want to have them as `staticmethod`? They could be just regular functions...

Reply:
> `_dict` is redundant, it's already in the type.

Sure





> Why do you want to have them as `staticmethod`? They could be just regular functions...

I agree. I wanted to do it this way just to avoid to import for all the protocols these two functions.
I don't if I should call it laziness or just a strategy to avoid some imports.

Reply:
I am also not sure why these need to be `@staticmethod`. Is there any routine where you use them with a `data_dict` that is not `self.data` and `self.global_params_dict`?